frontend/src/views/Lobby.tsx

```diff
@@ -193,12 +253,7 @@ function LobbyPage() {
       getRoom(location.state.roomId)
         .then((res) => {
           setStateFromRoom(res);
-          // Reset the user to hold the ID.
-          res.users.forEach((user: User) => {
-            if (user.nickname === location.state.user.nickname) {
-              setCurrentUser(user);
-            }
-          });
+          updateCurrentUserDetails(res.inactiveUsers);
```

Comment:
nit: could you change this to pass in the list of all users? I'm hoping that doesn't break anything in your PR but I remember that was needed to fix the CMD+SHIFT+T bug

Reply:
Yeah no problem -- I'm not certain whether that fixed the error on this page, but it doesn't break my PR so I made the change 👍